### PR TITLE
average predictive value difference metric implementation #376

### PR DESCRIPTION
**Pull request by sreeja-g**
#376 implementation

average predictive value difference metric

### CODE REVIEW

File: aif360/sklearn/metrics/metrics.py [hunk 1]
```diff
@@ -344,6 +344,27 @@ def specificity_score(y_true, y_pred, *, pos_label=1, sample_weight=None,
     return _prf_divide(tn, negs, 'specificity', 'negative', None,
                        ('specificity',), zero_division).item()
 
+def false_omission_score(y_true, y_pred, *, pos_label=1, sample_weight=None,
```
Comment (hoffmansc) [inline aif360/sklearn/metrics/metrics.py:347]: By [scikit-learn convention](https://scikit-learn.org/stable/modules/model_evaluation.html#defining-your-scoring-strategy-from-metric-functions), functions ending in `_score` should be maximized while functions ending in `_error` should be minimized. Can we change this name to `false_omission_rate_error`?

File: aif360/sklearn/metrics/metrics.py [hunk 2]
```diff
@@ -344,6 +344,27 @@ def specificity_score(y_true, y_pred, *, pos_label=1, sample_weight=None,
...
+            zero division. If set to “warn”, this acts as 0, but warnings are
+            also raised.
+    """
+    _check_zero_division(zero_division)
+    MCM = multilabel_confusion_matrix(y_true, y_pred, labels=[pos_label],
+                                      sample_weight=sample_weight)
+    tn, fn = MCM[:, 0, 0], MCM[:, 1, 0]
+    negs = tn + fn
+    return _prf_divide(fn, negs, 'negative_predictive', 'negative', None,
+                       ('negative_predictive',), zero_division).item()
```
Comment (hoffmansc) [inline aif360/sklearn/metrics/metrics.py:366]: The error message should read: ```UndefinedMetricWarning: False Omission Rate is ill-defined and being set to 0.0 due to no predicted negative samples. Use `zero_division` parameter to control this behavior.```

Therefore, this line should be:
```
    return _prf_divide(fn, negs, 'false omission rate', 'predicted negative',
                       None, ('false omission rate',), zero_division).item()
```

**Review by hoffmansc — CHANGES_REQUESTED** [review]
Great work! Just a couple small comments and we should be good to go!

**Review by hoffmansc — APPROVED** [review]
Looks great, thanks!
